apollo-router/src/error.rs

```diff
@@ -17,10 +17,12 @@ use thiserror::Error;
 use tokio::task::JoinError;
 use tracing::level_filters::LevelFilter;
 
+pub use crate::configuration::ConfigurationError;
 pub(crate) use crate::graphql::Error;
 use crate::graphql::Response;
 use crate::json_ext::Path;
 use crate::json_ext::Value;
+pub use crate::reload::Error as ReloadError;
```

Comment:
This makes me sad... This is part of the code we copied from `tracing_subscriber` and is another good reason why we should get rid of our hot reload for tracing stuff...

Reply:
Good news! https://github.com/apollographql/router/pull/1463 removes this type entirely